### PR TITLE
Use storybook syntax-highlighter

### PR DESCRIPTION
**Pull request by ndelangen**
This makes it so the code is theme-able from storybook 🎉 

But it does change the default visual look of the addon quite a bit.
It's the same as addon-source.. WDYT @stof ?

Published PR with canary version: `7.1.12-canary.92.253`

### CODE REVIEW

**Comment by stof** [comment]
@ndelangen this makes perfect sense to me.

Note that I'm not using addon-jsx myself (I'm using `@storybook/html` so I don't have any JSX at all). I contributed a PR on this repo mainly to get rid of the weekly github vulnerability report that I started receiving after being invited in the storybookjs organization.

File: package.json [hunk 1]
```diff
@@ -38,11 +38,10 @@
     "testURL": "http://localhost"
   },
   "dependencies": {
+    "@storybook/components": "^5.2.5",
     "copy-to-clipboard": "^3.0.8",
     "js-beautify": "^1.8.8",
-    "prismjs": "^1.15.0",
-    "react-element-to-jsx-string": "^14.0.3",
-    "storybook-chromatic": "^3.0.3"
```
Comment (hipstersmoothie) [inline package.json]: Is this even needed? doesn't seem to be used anywhere

Comment (ndelangen) [reply package.json]: It's used to run chromatic locally, to set it up, can be removed once it's automated in a CI / action (which is now is)

**Comment by ndelangen** [comment]
@hipstersmoothie shall we do a minor release?

**Comment by hipstersmoothie** [comment]
:rocket: PR was released in v7.1.12 :rocket:

**Pull request merged**